org.eclipse.xtext.xtext.generator/src/org/eclipse/xtext/xtext/generator/junit/Junit4Fragment2.xtend

```diff
@@ -27,11 +27,19 @@ class Junit4Fragment2 extends AbstractStubGeneratingFragment {
 			"org.eclipse.xtext.testing"
 	}
 	
+	def protected getXbaseTestingPackage() {
+		if (useDeprecatedClasses)
+			"org.eclipse.xtext.xbase.junit"
+		else
+			"org.eclipse.xtext.xbase.testing"
+	}
+	
 	override generate() {
 		if (projectConfig.runtimeTest.manifest !== null) {
 			projectConfig.runtimeTest.manifest => [
 				requiredBundles.addAll(
 					testingPackage,
+					xbaseTestingPackage,
```

Comment:
The flag would allow you to SKIP such dep, not to add it.

The dep would be added by default, mimicking the default deps of .tests project created by the Xtext wizard. If the mwe2 adds the dep by default, as I was proposing, then mwe2 will bring an existing .tests project (created with Xtext 2.10) to the same situation of .tests project freshly created by the Xtext wizard.

All in all, my proposal to add this dep by default when running mwe2 aims at easier porting existing projects (Xtext 2.10, not necessarily based on Xbase) to Xtext 2.11.